Library/Homebrew/attestation.rb

```diff
@@ -104,6 +107,11 @@ def self.check_attestation(bottle, signing_repo, signing_workflow = nil, subject
         # Even if we have credentials, they may be invalid or malformed.
         raise GhAuthNeeded, "invalid credentials" if e.status.exitstatus == 4
 
+        if Formula["gh"].any_installed_version.version < GH_ATTESTATION_MIN_VERSION
```

Comment:
We should probably check `gh --version` instead. `gh_executable` may or may not be installed with `brew`.

Reply:
That would be better, but we currently don't verify attestations unless `gh` is installed from Homebrew. See here:
https://github.com/Homebrew/brew/blob/2d26c89e81e838f34927c89d9d368765e9dc56df/Library/Homebrew/attestation.rb#L52

I'll make the change though!

Reply:
We do _if_ the user explicitly enables attestations via `HOMEBREW_VERIFY_ATTESTATIONS` 🙂 -- so yeah, we do need this to use `--version` instead.

Reply:
Ah, my bad – updating this. Thanks!